templates/cicd/pipeline_cfn.yml

```diff
@@ -0,0 +1,291 @@
+#  Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+#  the License. A copy of the License is located at
+#      http://aws.amazon.com/apache2.0/
+#  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#  CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and
+#  limitations under the License.
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CodePipeline for the {{$.ProjectName}}
+Parameters:
+  // TODO: #273 make this source-provider-agnostic
```

Comment:
huh does this comment render with the `//` don't we need to change it to `#`?

Reply:
Yea I noticed that. Already updated in the PR I'm pushing out today.